### DISCUSSION TITLE
Printer is not seeing any of my SD's or stock USB.

### DISCUSSION THREAD

**Discussion by levleon13** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2253] [Q&A]
Sidewinder x2 ( flashed screen only ) 
tried to format my cards to all types, none worked. just shows blank screen once in.
i have downloaded a version from here:
https://3dprintbeginner.com/sidewinder-x2-custom-touchscreen-firmware/
and i guess it is outdated and theres a ver with a fix for that, but i am a bit lost figuring out which files i need to put on my card to flash it to the latest, the ones here are not similar to the rar i downloaded from 3dprintbeginner website.
Pointing out to the correct files for my X2 tft display would be really appreciated! 
currently it says MKS TFT28 v1.0
Firmware: Diga-tech v1.0.27.x oct 26 2021

**Comment by kisslorand** [comment]
How do you intend to upload a new firmware to your TFT?

**Reply by levleon13** [reply]
as i did with the older version, SD card. Updating fw works.

**Comment by kisslorand** [comment]
Can you check if a build before 10th of November works for you?

**Reply by levleon13** [reply]
> Can you check if a build before 10th of November works for you?

i'd check if i knew which files to pick to not brick my display
